src/pages/joinGameLobby/[id].js

```diff
@@ -99,24 +119,48 @@ export default function Lobby() {
 
   useEffect(() => {}, [count]);
   useEffect(() => {
+    //if they are not Host
     if (getCookie("isHost") === "false") {
       setButtonDisabled(true);
+    } else{
+      //add a POST request to update users in lobby
     }
+
+    const urlCode = window.location.pathname.split('/');
+    console.log(urlCode[urlCode.length -1])
     setGameLobbyText(urlCode[urlCode.length -1])
+    getUserName();
```

Comment:
you have access to a router.query value here because it's a dynamic route; "router.query.id" should give you the same info since it's inside the useEffect(meaning the component has mounted). You'll also need this value to create a game and navigate to the correct game screen when someone/the host clicks "start"

Reply:
Yeah I was using the {pid} from router.query and it was giving me trouble. Its still in the top but it was undefined when I was messing with it last night.